container/root/etc/apache2/sites-available/000-docker.conf

```diff
@@ -4,4 +4,15 @@
   LogLevel info
   ErrorLog /dev/stdout
   CustomLog /dev/stdout combined
+
+  <Directory /var/www/html>
+    # One rule to rule them all.
+    RewriteEngine On
+    RewriteBase /
+
+    RewriteCond %{REQUEST_FILENAME} !-f
```

Comment:
interestingly, if we do this, we can disable the options override -- granting us a "performance" boost. what do you think?